### PR TITLE
Created alias.md for the reference page

### LINKED ISSUE #238: create a reference page for 'alias'

### PR DESCRIPTION
**Pull request by Anthuang**
# Advanced HW 7 submission

Closes #238 

Name: Anthony Huang
Uniqname: anthuang

### CODE REVIEW

**Comment by mterwill** [comment]
Looks like you have some Markdown formatting issues:

image

File: _commands/basics/alias.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+bash: cd: Desktop: No such file or directory
+~~~
+
+### Making aliases permanent
+If you just run the alias command from the shell, it would last for the duration of the shell (until the shell is closed). To make the aliases permanent, we can just tell the shell to run these commands when the shell starts.
+
+#### Beginner
+For people who are relatively new to this, we can do this easily. Careful, for people with code already in the .bashrc file, you should instead use an editor to edit .bashrc. So, instead of "cat >" you should use "vim" or your editor of choice.
+~~~bash
+$ cat > ~/.bashrc
```
Comment (mterwill) [inline _commands/basics/alias.md]: Why don't you make this an append (`>>`) operator so it won't overwrite by default?

File: _commands/basics/alias.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+### Useful Options / Examples
+
+#### Usage
+Things to note: when using alias, make sure that there is no space around the equal sign.
+~~~bash
+$ alias l = "ls"
+alias l='ls -CF'
+bash: alias: =: not found
+~~~
```
Comment (mterwill) [inline _commands/basics/alias.md]: You just need to add a bunch of newlines in between text + code blocks and it should render better.

Comment (Anthuang) [inline _commands/basics/alias.md]: Okay. I added all the newlines. I was looking at the markdown file through github and didn't see any formatting issues so I thought the formatting had no problem.

File: _commands/basics/alias.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+### Useful Options / Examples
+
+#### Usage
+
+Things to note: when using alias, make sure that there is no space around the equal sign.
+
+~~~bash
+$ alias l = "ls"
+alias l='ls -CF'
```
Comment (mterwill) [inline _commands/basics/alias.md]: This is syntactically correct – why is there an error below it?

File: _commands/basics/alias.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+~~~
+
+This shows that you can alias commands with multiple words.
+
+#### Example
+
+~~~bash
+$ alias gdesk="cd ~/Desktop"
+$ pwd
+/home/user/Desktop
```
Comment (mterwill) [inline _commands/basics/alias.md]: Creating an alias would not actually run the command – could you clarify this example?

File: _commands/basics/alias.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+~~~
+
+One thing to be careful about is if you alias the cd command without using the ~, your alias might not always work. For example:
+
+~~~bash
+$ alias gdesk="cd Desktop"
+bash: cd: Desktop: No such file or directory
```
Comment (mterwill) [inline _commands/basics/alias.md]: Same here.

File: _commands/basics/alias.md (hunk 2 above)
Comment (mterwill) [inline _commands/basics/alias.md]: Formatting looks good now, thanks!

File: _commands/basics/alias.md (hunk 3 above)
Comment (Anthuang) [inline _commands/basics/alias.md]: Ah, I was just trying to show that you can't have spaces around the "=".

**Comment by mterwill** [comment]
Cool! LGTM. Thanks for fixing those up. Merging + giving you credit.

**Pull request merged**